impl/shim/mapreduce/src/main/java/org/pentaho/big/data/impl/shim/mapreduce/MapReduceServiceImpl.java

```diff
@@ -89,7 +89,14 @@ public MapReduceJobSimple executeSimple( URL resolvedJarUrl, String driverClass,
 
     final List<String> finalClassNamesInJarWithMain = classNamesInJarWithMain;
 
-    final Class<?> mainClassFromManifest = jarUtility.getMainClassFromManifest( resolvedJarUrl, classLoader );
+    Class<?> mainClassFromManifest = null;
+    try {
+      mainClassFromManifest = jarUtility.getMainClassFromManifest( resolvedJarUrl, classLoader );
+    } catch ( Exception e ) {
```

Comment:
Should JarUtility more gracefully handle cases where it can't load?  I hate seeing a generic catch block be part of normal execution flow.